packages/type-jsonrpc/src/state.ts

```diff
@@ -19,6 +19,57 @@ const call: RpcMethodOpt = {
   type: 'Bytes'
 };
 
+const getChildKeys: RpcMethodOpt = {
```

Comment:
Can we align the ordering here to match what is in substrate? It looks very arbitrary, but it helps an immense amount when going through and looking at discrepancies and missing methods - there are a lot. 

(For the exports below, right at the end, we do it alphabetical - so it is sane)